### PR TITLE
Added timedelta field conversion

### PR DESCRIPTION
**Pull request by creynold**
* Converts timedelta to/from total_seconds
  * Added unit tests for these functions.

### CODE REVIEW

**Comment by jwhitlock** [comment]
Thanks!  A few things to fix:

- [ ] Make flake8 happy with docstrings (can run ``make qa`` to check locally)
- [ ] Add a fallback for Python 2.6, which lacks ``timedelta.total_seconds()``.  There's sample code at https://github.com/BVLC/caffe/pull/93/files for the calculation.  I'd want it a ``try / except AttributeError`` block, so ``total_seconds`` is used if present, and with a comment that it's a Python 2.6 fallback, so we can drop it when we drop Python 2.6 support.
- [ ] Convert to/from a string if there are fractional seconds, to avoid weirdness around serialization of floats (see ``field_datetime_to_json``/``field_datetime_to_json``)
- [ ] Add your info to AUTHORS.rst

File: drf_cached_instances/cache.py [hunk 1]
```diff
@@ -265,6 +265,36 @@ def field_datetime_to_json(self, dt):
...
+
+        If there are fractions of a second the return value will be a
+        string, otherwise it will be an int.
+        """
+        try:
+            if td.microseconds > 0:
+                return str(td.total_seconds())
+            else:
+                return int(td.total_seconds())
+        except AttributeError:
```
Comment (jwhitlock) [inline drf_cached_instances/cache.py]: Changing to this will make coveralls happier:

```
except AttributeError:  # pragma: no cover
```

**Comment by jwhitlock** [comment]
This looks good.  Do you want to keep updating until coveralls is happy, or do you want me to do final cleanup?

File: drf_cached_instances/cache.py (hunk 1 above)
Comment (creynold) [inline drf_cached_instances/cache.py]: Ah, I was wondering how to either test that section or let it ignore it.

**Comment by creynold** [comment]
I just added the `#pragma` statement, so hopefully that covers it.

**Comment by creynold** [comment]
Sorry, I ran `make qa` but didn't see this in there. Apologies. By the way is there a way to make flake8 not check the site-packages?

**Comment by jwhitlock** [comment]
A few come to mind:
1) Don't put site-packages in your repository folder (Mine are at ~/.virtualenvs/drf-cached-instances)
2) Create a global flake8 config: http://flake8.readthedocs.org/en/2.4.1/config.html#global , something like:

```
[flake8]
exclude = site-packages/*
```

Your paths may vary...

**Comment by creynold** [comment]
Ok, I just moved my virtualenv. Now I'm not sure why flake8 is complaining about class docstrings since I didn't change anything about them.

**Comment by jwhitlock** [comment]
There's a new version of pep257 released (0.6.0 -> 0.7.0), and TravisCI is installing the new one.  I'll fix this after merging.

**Pull request merged**